File: src/DutyFree.Entity/ValueType/Time.cs
namespace DutyFree.Entity.ValueType
{
    public class Time
    {
        public int Hour { get; set; }
        public int Minute { get; set; }
        public int Second { get; set; }

        public Time(int hour, int minute, int second)
        {
            this.Hour = hour;
            this.Minute = minute;
            this.Second = second;
        }

        public int GetSeconds()
        {
            return Hour * 3600 + Minute * 60 + Second;
        }
    }
}

File: bomeiboyou/src/entity/time.ts
export class Time {
    public constructor(public hour: number, public minute: number, public second: number) { }

    public getTime(): string {
        return ("0" + this.hour).slice(-2) + ":" + ("0" + this.minute).slice(-2) + ":" + ("0" + this.second).slice(-2);
    }
}